acceptance/upgrade/suite_test.go

```diff
@@ -0,0 +1,83 @@
+package upgrade_test
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/epinio/epinio/acceptance/helpers/proc"
+	"github.com/epinio/epinio/acceptance/testenv"
+	"github.com/epinio/epinio/internal/cli/settings"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+func TestAcceptance(t *testing.T) {
+	RegisterFailHandler(FailWithReport)
+	RunSpecs(t, "Acceptance Suite for Application Tests")
+}
+
+var (
+	nodeSuffix, nodeTmpDir string
+
+	// serverURL is the URL of the epinio API server
+	serverURL, websocketURL string
+
+	env testenv.EpinioEnv
+)
+
+var _ = BeforeSuite(func() {
+	fmt.Printf("Running tests on node %d\n", GinkgoParallelProcess())
+
+	testenv.SetRoot("../..")
+	testenv.SetupEnv()
+
+	nodeSuffix = fmt.Sprintf("%d", GinkgoParallelProcess())
+	nodeTmpDir, err := ioutil.TempDir("", "epinio-"+nodeSuffix)
+	Expect(err).NotTo(HaveOccurred())
+
+	out, err := testenv.CopyEpinioSettings(nodeTmpDir)
+	Expect(err).ToNot(HaveOccurred(), out)
+	os.Setenv("EPINIO_SETTINGS", nodeTmpDir+"/epinio.yaml")
+
+	config, err := settings.LoadFrom(nodeTmpDir + "/epinio.yaml")
+	Expect(err).NotTo(HaveOccurred())
+
+	env = testenv.New(nodeTmpDir, testenv.Root(), config.User, config.Password)
+
+	out, err = proc.Run(testenv.Root(), false, "kubectl", "get", "ingress",
+		"--namespace", "epinio", "epinio",
+		"-o", "jsonpath={.spec.rules[0].host}")
+	Expect(err).ToNot(HaveOccurred(), out)
+
+	serverURL = "https://" + out
+	websocketURL = "wss://" + out
```

Comment:
We keep copying these 2 around in all suites but I only see them being used in 2 tests:
- `acceptance/api/v1/application_logs_test.go`
- `acceptance/api/v1/application_exec_test.go`

Reply:
hmm the [linter](https://github.com/epinio/epinio/runs/7180785143?check_suite_focus=true#step:6:33) is not happy!
```
Error: `serverURL` is unused (deadcode)
```

Reply:
My bad... I know